inception/inception/inception_distributed_train.py

```diff
@@ -110,7 +114,7 @@ def train(target, dataset, cluster_spec):
   is_chief = (FLAGS.task_id == 0)
 
   # Ops are assigned to worker by default.
-  with tf.device('/job:worker/task:%d' % FLAGS.task_id):
+  with tf.device('/job:worker/task:%d/gpu:%d' % (FLAGS.task_id, FLAGS.gpu_id)):
```

Comment:
`tf.device('/job:worker/task:%d' % FLAGS.task_id)` is better for client code. So maybe it is time to modify Tensorflow code. Should I post an issue to Tensorflow repo ?